testsuite/driver/runtests.py

```diff
@@ -81,7 +81,8 @@ def signal_handler(signal, frame):
     config.only = args.only
     config.run_only_some_tests = True
 
-with open("../skip-tests.txt", 'r') as f:
+from os.path import dirname, abspath
+with open(os.path.join(dirname(dirname(abspath(__file__))), "skip-tests.txt"), 'r') as f:
```

Comment:
Does this matter, because we intended to revert the changes to this file for the merge.

Reply:
This allows to run `make test` (or `make accept`) in `linear` directory, which I found useful and I think will be used a few more times in the future.